src/lib/Hydra/Schema/BuildProducts.pm

```diff
@@ -99,8 +94,6 @@ __PACKAGE__->add_columns(
   { data_type => "text", is_nullable => 0 },
   "filesize",
   { data_type => "bigint", is_nullable => 1 },
-  "sha1hash",
-  { data_type => "text", is_nullable => 1 },
```

Comment:
I wonder if we should keep this column and continue serving the data, but null on all future builds?

Reply:
I've removed the upgrade script to we don't delete any data.